README.md

```diff
@@ -100,7 +108,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is how we want to represent our objects as tables in the database. The objects attributes are the columns in the table. We use primary keys and foreign keys to represent on to many relationships in sql. If we use the person one to many relationship with wishes we can say each person has a unique primary key. But a person can have many wishes so each wish will have a foreign key which tells us which person that wish belongs to.
```

Comment:
A record's ID is its primary key -- it's not necessarily related to associations.